apps/ledger-live-mobile/src/components/FirebaseFeatureFlags.tsx

```diff
@@ -38,8 +47,40 @@ export const getAllDivergedFlags = (): { [key in FeatureId]: boolean } => {
   return res;
 };
 
-export const FirebaseFeatureFlagsProvider = ({ children }: Props) => (
-  <FeatureFlagsProvider getFeature={getFeature}>
-    {children}
-  </FeatureFlagsProvider>
-);
+export const FirebaseFeatureFlagsProvider = ({ children }: Props) => {
+  const [localOverrides, setLocalOverrides] = useState({});
+
+  const overrideFeature = (key: FeatureId, value: Feature): void => {
+    const actualRemoteValue = getFeature(key);
+    if (!isEqual(actualRemoteValue, value)) {
+      const overridenValue = { ...value, overridesRemote: true };
+      setLocalOverrides(currentOverrides => ({
+        ...currentOverrides,
+        [key]: overridenValue,
+      }));
+    }
+  };
+
+  const resetFeature = (key: FeatureId): void => {
+    setLocalOverrides(currentOverrides => ({
+      ...currentOverrides,
+      [key]: undefined,
+    }));
+  };
+
+  // Nb wrapped because the method is also called from outside.
+  const wrappedGetFeature = useCallback(
```

Comment:
If I'm not wrong, we don't need this since `getFeature` isn't defined inside the component

Reply:
@gre took it out for something, that's why I had to do this 🤔